tests/flow/test_concurrent_query.py

```diff
@@ -331,3 +332,22 @@ def test_10_write_starvation(self):
         # delete the key
         self.conn.delete(GRAPH_ID)
 
+    def test_11_concurrent_resize_zero_matrix(self):
+        self.graph = Graph(self.conn, GRAPH_ID)
+
+        self.graph.query("CREATE (:N)")
+
+        def resize_and_query():
+            g = redis.commands.graph.Graph(self.conn, GRAPH_ID)
+
+            for j in range(1, 10):
+                g.query("UNWIND range(1, 10000) AS x CREATE (:M)")
+                for i in range(1, 10):
+                    g.query("MATCH (n:N)-[r:R]->() RETURN r")
+        
+        loop = asyncio.get_event_loop()
+        tasks = []
+        for i in range(1, 10):
+            tasks.append(loop.create_task(asyncio.to_thread(resize_and_query)))
```

Comment:
`to_thread` actually invokes a thread? or do all tasks run on the same loop?
if so I don't see how are we achieving concurrency here